src/h5web/dataset-visualizer/VisDisplay.tsx

```diff
@@ -15,13 +15,10 @@ interface Props {
 
 function VisDisplay(props: Props): JSX.Element {
   const { activeVis, dataset } = props;
-  const { Component: VisComponent } = VIS_DEFS[activeVis];
-
   const datasetDims = isSimpleShape(dataset.shape) ? dataset.shape.dims : [];
-  const [mapperState, setMapperState] = useState<DimensionMapping | undefined>(
-    undefined
-  );
```

Comment:
`useState` uses `undefined` by default, and when you write `useState<DimensionMapping>()`, the implied type is `DimensionMapping | undefined`.